src/Microsoft.VisualStudio.ProjectSystem.Managed/ProjectSystem/Rules/PropertyPages/ApplicationPropertyPage.VisualBasic.xaml

```diff
@@ -246,6 +246,14 @@
         </NameValuePair.Value>
       </NameValuePair>
     </StringProperty.Metadata>
+    <StringProperty.ValueEditors>
+      <ValueEditor EditorType="String">
+        <ValueEditor.Metadata>
+          <NameValuePair Name="EvaluatedValueValidationRegex" Value="^\d*$" />
+          <NameValuePair Name="EvaluatedValueFailedValidationMessage" Value="This property must be an integer number." />
```

Comment:
This value will need to be localised. I think we have a way to configure this with the Xliff tools stuff. Maybe @adamint can point us at that.

Reply:
thanks for the observation, done in the newest commit.